src/components/index.js

```diff
@@ -24,6 +24,8 @@ import VisConfigSliderFactory from './side-panel/layer-panel/vis-config-slider';
 import LayerConfigGroupFactory from './side-panel/layer-panel/layer-config-group';
 import {ChannelByValueSelectorFactory} from './side-panel/layer-panel/layer-configurator';
 import {appInjector} from './container';
+import FieldSelectorFactory from './common/field-selector';
```

Comment:
nitpick, move the import above `appInjector`

Reply:
Done